game_eggs/steamcmd_servers/frozen_flame/README.md

```diff
@@ -0,0 +1,18 @@
+# Frozen Flame Dedicated Server
+
+Frozen Flame is a Survival RPG set in the vast world of Arсana, an ancient land once governed by the Dragons.
+
+## Server Ports
+
+At this moment you cannot change the server ports. These are the default ports:
```

Comment:
are only these ports supported? I cant test it, because i dont know the game. But i cant believe, that only these ports are supported

Reply:
Oh i see. I will change this

Reply:
@gOOvER Ports are now configurable.